Add docs for delegatedPermissionClassifications and permissionGrantPolicies (#10183)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/permissiongrantpolicy-post-permissiongrantpolicies.md

---

> title: "Create a new permissionGrantPolicy"

Reviewer: ```suggestion
title: "Create a permissionGrantPolicy"
```

> description: "Creates a new permissionGrantPolicy describing the conditions under which permissions may be granted."

Reviewer: ```suggestion
description: "Creates a permissionGrantPolicy that describes the conditions under which permissions may be granted."
```

localization_priority: Normal
doc_type: apiPageType
ms.prod: "microsoft-identity-platform"
author: "psignoret"
---

# Create a permissionGrantPolicy

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Creates a new [permissionGrantPolicy](../resources/permissiongrantpolicy.md). A permission grant policy is used to describe the conditions under which permissions can be granted (for example, during application consent).

Reviewer: ```suggestion
Creates a [permissionGrantPolicy](../resources/permissiongrantpolicy.md). A permission grant policy is used to describe the conditions under which permissions can be granted (for example, during application consent).
```

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type      | Permissions (from least to most privileged)              |
|:--------------------|:---------------------------------------------------------|
|Delegated (work or school account) | PermissionGrantPolicy.ReadWrite.All |
|Delegated (personal Microsoft account) | Not supported.    |
|Application | PermissionGrantPolicy.ReadWrite.All |

## HTTP request

<!-- { "blockType": "ignored" } -->
```http
POST /policies/permissionGrantPolicies
```

## Request headers

| Name       | Description|
|:-----------|:----------|
| Authorization | Bearer {token}. Required.  |
| Content-type | application/json. Required. |

## Request body

In the request body, supply a JSON representation of an [permissionGrantPolicy](../resources/permissiongrantpolicy.md) object.

## Response

If successful, this method returns a `201 Created` response code and a [permissionGrantPolicy](../resources/permissiongrantpolicy.md) object in the response body.

## Examples

### Request

Here is an example of the request.

> # [HTTP](#tab/http)

Reviewer: Please remove line 58.

<!-- {
  "blockType": "request",
  "truncated": true,
  "name": "create_permissiongrantpolicy"
}-->

```http
POST https://graph.microsoft.com/beta/policies/permissionGrantPolicies
Content-Type: application/json

> Content-Length: 110

Reviewer: Please remove line 69.

{
  "id": "my-custom-consent-policy",
  "displayName": "Custom application consent policy",
  "description": "A custom permission grant policy to customize conditions for granting consent."
}
```

> ---

Reviewer: Please remove line 78.

### Response

> Here is an example of the response.

Reviewer: ```suggestion
The following is an example of the response.
```
